lib/tasks/user.rake

```diff
@@ -0,0 +1,38 @@
+namespace :hyku do
+  namespace :superadmin do
+    desc 'Create an initial superadmin user'
+    task create: :environment do
+      puts 'Creating an initial superadmin user.'
+      user = prompt_to_create_user
+      user.add_role(:superadmin)
+      puts 'User created.'
+    end
+
+    def prompt_to_create_user
```

Comment:
So, am I reading this PR right that we now have two rake tasks that can elevate an existing user to superadmin? I'm surprised the new task works for existing users given we already have the above rake task for that. Maybe I'm misreading this.